extensions/amp-form/0.1/amp-form.js

```diff
@@ -1052,14 +1052,13 @@ export class AmpForm {
                   /* detail */ null,
                   {bubbles: true});
               container.dispatchEvent(renderedEvent);
+              this.resources_.mutateElement(container, () => {});
```

Comment:
I think we should wrap the `appendChild` call in the `mutateElement` callback. It probably should have been that way to begin with.

Reply:
done. thanks Carlos.

Reply:
@jridgewell to verify that that `appendChild` should be wrapped in `mutateElement`